source/manual/manage-assets.html.md.erb

```diff
@@ -125,6 +125,6 @@ changing the URL, follow these steps:
    ```
 
    ```ruby
-   attachment = Attachment.find(asset-id-from-url) # e.g. 123456
-   attachment.attachment_data.update(file_size: 123)
+   attachment = Attachment.find_by(attachment_data_id: asset-id-from-url) # e.g. 123456
```

Comment:
Oops!  I must've been doing this by trial and error, then copied the wrong thing 🤦